### PR TITLE
Update ItemsRepeater Animated Scrolling with Content Sample Behaviour

### LINKED ISSUE #376: Behavior of Animation Scrolling and Content display buttons in ItemsRepeater sample page is confusing
**Describe the bug**

When using touch narrator to scroll items in this sample, it may be difficult for users to understand the behavior they're seeing. The color swatch changes based on which item is centered in the ItemsRepeater, but it may not be obvious what's causing the color to change.

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery and turn on narrator.
2. Activate Items Repeater from All Controls Page.
3. Navigate to first button present under "Animated Scrolling and Content Display" and swipe right and check if color changes or not,swipe right 4-5 times and simultaneously observe if color is changing or not.

**Expected behavior**

Color should not change on item navigation and it should change only when its activated with double tap.

**Actual behavior**

For 1st few buttons the color doesn't change but when we move forward the color changes but its not in sync .

**Screenshots**

image

**Version Info**

Gallery version (found on Settings page): 

1.2.13.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/25469691)

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

This sample was previously updating content sample colour on GotFocus and on scroll, causing confusions especially with keyboard navigation and Narrator.  Colour selection is now updated to only on click.

## Motivation and Context

Fixes #376 and #378

## How Has This Been Tested?

Manual Testing

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 1]
```diff
@@ -242,13 +242,12 @@ the following attributes: Height, MaxHeight, Length, MaxLength, Diameter, and Ma
...
                               Height="175"
-                              Width="250"
-                              ViewChanged="Animated_ScrollViewer_ViewChanged">
+                              Width="250">
                     <muxc:ItemsRepeater x:Name="animatedScrollRepeater">
                             <DataTemplate x:DataType="x:String">
                             <Button Content="{x:Bind}"
                                         Background="{x:Bind}"
-                                        Click="Animated_GotItem" 
+                                        Click="Animated_ClickItem"
```
Comment (ranjeshj) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml:250]: nit: OnAnimatedItemClicked

**Comment by ranjeshj** [comment]
@chigy Is this acceptable accessibility behavior for this sample ?

**Comment by chigy** [comment]
@ranjeshj , @karenbtlai , am I correct that the change is you are only updating the sample color once user action on the list item?

**Comment by karenbtlai** [comment]
> @ranjeshj , @karenbtlai , am I correct that the change is you are only updating the sample color once user action on the list item?

Yup. Before everything was on GotFocus, which was causing confusion especially when using keyboard navigation. Now Click and GotFocus events are decoupled.

**Review by ranjeshj — APPROVED** [review]

**Pull request merged**